src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_moving_columns.scss

```diff
@@ -431,4 +439,11 @@ A list of alert messages to notify users in the moving column
     .moving-column-alerts {
         display: none;
     }
+
+    .moving-columns[class^="is-show-show"],
+    .moving-columns[class*=" is-show-show"] {
+        .moving-column.is-show:first-child {
+            display: none;
+        }
+    }
```

Comment:
This will break in the following case: 

-   have a small window (only one column)
-   navigate to the comment column
-   uncollapse the first column
-   print

This will only print the second column while you only see the first one. The reason for this is that the moving column state is actually `is-show-show-hide` but `focus` is set to the first column.

Note that moving column state and focus are not well defined and are likely to change.

I am also not sure whether this is a generic solution that should be in core.

My suggestion would be to add a `has-focus` state to moving columns via JavaScript and then only show the focused column on print.